mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -45,6 +50,8 @@ def SYCL_Dialect : Dialect {
 class SYCL_Op<string mnemonic, list<Trait> traits = []>
     : Op<SYCL_Dialect, mnemonic, traits>;
 
+// methodNames MUST be a sorted list as the interface verification uses binary
```

Comment:
As we are now using std::find, can this requirement be removed?

Reply:
Indeed. Removed!